bridge-adapters/kinetic-bridgehub-adapter-generic-rest/README.md

```diff
@@ -0,0 +1,66 @@
+# Kinetic Bridgehub Adapter Generic Rest
+This adapter can be used to make generic rest calls that require basic auth.  Using JSON Path the adapter is able to handle JSON responses.
+___
+## Adapter Configurations
+Name | Description
+------------ | -------------
+Username | Username of user to the source system
+Password | Privileged user's password
+URL Origin | Web address to AR System server (ex: https://foo.bar.com)

```

Comment:
Description can't be to a specific type of system. Maybe give a couple of examples to real systems instead of foo.bar.com.